src/app/pages/health-declaration/health-declaration-routing.module.ts

```diff
@@ -1,17 +1,31 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+// eslint-disable-next-line max-classes-per-file
```

Comment:
is this lint-disable nessacary?

Reply:
Yeah as the nav guard is now done inline within the routing module, the linting will complain about there being more than 1 class present